src/app/containers/FrontPageSection/__snapshots__/index.test.jsx.snap

```diff
@@ -153,6 +153,7 @@ exports[`FrontPageSection Container snapshots should render correctly for canoni
         },
       }
     }
+    visuallyHidden={false}
```

Comment:
This snapshot has `visuallyHidden={false}` 3 times, but no `visuallyHidden={true}`, howcome? I would expect the first to be `visuallyHidden={true}`

Reply:
Hadn't specified the `sectionNumber`. `sectionNumber` is now specified as 0 for the test cases, as there is only one `group` in each of test data